src/main/java/org/fogbowcloud/manager/core/plugins/identity/ldap/LdapIdentityPlugin.java

```diff
@@ -99,8 +103,7 @@ public Token createToken(Map<String, String> userCredentials) {
 			name = ldapAuthenticate(userId, password);
 		} catch (Exception e) {
 			LOGGER.error("Couldn't load account summary from LDAP Server.", e);
-			Integer statusResponse = HttpStatus.SC_UNAUTHORIZED;
-			throw new RuntimeException(statusResponse.toString());
+			throw new InvalidCredentialsException("Couldn't load account summary from LDAP Server.");
```

Comment:
sometimes people catch an exception and pass it as an argument to the more specific one. E.g, passing the "e" reference to the InvalidCredentialsException. In this case, there is no need to log. Note that, it is very likely we are going to have two log lines related to this error, one here and another in the code that is going to catch the InvalidCredentials.

I'm not against the current solution but take some time to analyze the trade-offs.